### ISSUE TITLE
Slimmer templates

### ISSUE THREAD

**Issue by LogicParrot** [cuberite/cuberite#3313]
Here is our current template. In my opinion, this is pure annoyance and inefficiency.

```
Client version:   x.x.x
Server OS:        Windows/Linux/macOS/BSD/...
Commit id:        (Found at the top of the server's console output)


### Expected behavior


### Actual behavior


### Steps to reproduce the behavior

```

**Case in point**

Here's a typical issue: #3312 

No template:
```
You can't deconstruct the Netherfences, they just disappear.
```

With the template:
```
Client version:   1.8.1
Server OS:        Linux
Commit id:        SPOAKOPKDPOK


### Expected behavior
I expect the nether fences to turn into pickable items.

### Actual behavior
They disappear.

### Steps to reproduce the behavior
Destroy a fence.
```

That's about 4 times more characters, with absolutely no extra useful info, the fix is a one liner: https://github.com/cuberite/cuberite/commit/baefed80618a326b7fd0bc492f587768a9fd2350

- Server OS is almost always irrelevant
- Client version is almost always irrelevant
- On the rare occasions where these are relevant, we can just ask the issue reporter
- The first two sections are almost always a repetition that can be clumped into one paragraph, and when we force it on people they just repeat themselves twice.
- The third section is *sometimes* useful but in most cases reproduction is trivial and doesn't justify its own paragraph.
- Commit ID is useful. If it were up to me the template would only contain `commit ID ` and a note like: `If reproduction is tricky, please explain how to reproduce this`.

**In summary**
To spare the reporters writing time and spare the devs reading time, I think we should get rid of this, and we definitely shouldn't **enforce it**.

**Comment by jammet** [comment]
This does sum it up pretty well. While I absolutely understand the need for details, and don't want to see us arguing over trivialities -- usually, when details become relevant, they're provided soon. I prefer the simple beauty of: "What seems to be the problem?" for

**Comment by sphinxc0re** [comment]
It is very important to have all the information above to eliminate Issues that are out of date. Sometimes a user doesn't know an Issue has already been fixed. Using the Template we can eliminate these false friends. We also can eliminate (Yes I like that word) Issues regarding compatibility to Minecraft versions and the OS is of course not irrelevant, because there are many OS specific failures that can be  detected that way. @cuberite/core-team what do you think?

**Comment by madmaxoft** [comment]
I think the template is good. Usually the first questions asked in an issue is "what server version are you using? Windows or Linux? What client?"; if I can spend less time asking these and more time actually coding or debugging, it's good. We're trading a bit of reporters' comfort for a lot of developers' comfort. 

Yes, there are issues for which the template is needlessly verbose. I wouldn't enforce it for blindingly obvious issues, but then, which reporter can actually distinguish these? If you are so sure as not to use it, and I find myself asking you those questions, I *will* mock you, at least in my head if not out loud :P

**Comment by LogicParrot** [comment]
Like I said, commit ID is often useful. 
What bother me the most are the 3 sections, which are 3 repititions 99.999% of the time.

**Comment by LogicParrot** [comment]
>I think the template is good. Usually the first questions asked in an issue is "what server version are you using? Windows or Linux? What client?"

Ok. Maybe I don't feel that as often, because I mainly work with AI, which typically doesn't care about OS/client version. But the 3 sections are my main issue.

**Comment by madmaxoft** [comment]
You would be surprised how often people are unable to describe their problems, or see a problem where others don't see one. That's the main reason for the three sections. The description above should be only a summary of the three sections.
Of course there are a lot of issues that don't need to be that verbose.
